experimental/dds/tree2/src/test/feature-libraries/editable-tree/editableTree.binder.spec.ts

```diff
@@ -537,6 +531,13 @@ describe("editable-tree: data binder", () => {
 					},
 					{ field: fieldSequencePhones, index: 0 },
 				],
+				[
```

Comment:
Hey Dragos, this change is because we now must insert each of the nodes one by one which triggers an additional notification here, at index 1. There are plans to fix this so it'll go back to the previous behavior.

Reply:
Acknowledged. Thank you!